crystalBox.py

```diff
@@ -236,13 +270,31 @@ def get_scatterers(self):
 
         print('getting scatterers')
         scatterers = self.scatterers
-        print(scatterers)
+        #print(scatterers)
         scatterers = [atm.split(' ') for atm in list(scatterers)]
         scatterers = [[val if val.isalpha() else float(val) \
                        for val in scatterer] for scatterer in scatterers]
         print(scatterers)
         return scatterers
     
+    def set_scatterers(self):
+        scatterers = self.estimate_uiso()
```

Comment:
This seems like it should be the place to generalise so that any crystal structure parmaeter can be modified. 

We should retain `estimate_uiso` as way to easily update only uisos using our standardised method.

Following comment above, setScatterers should automatically call process crystal to update FSq/Int to reflect the changes. 

Lastly, I try to systematically use "camelCase" so function name should be `setScatterers` (and I need to also update the copy/paset `get_Scatterers` that I stole from Zach)